base_agent/loco_mc_agent.py

```diff
@@ -218,24 +218,23 @@ def step(self):
         self.maybe_dump_memory_to_dashboard()
 
     def task_step(self, sleep_time=0.25):
-        # Clean finished tasks
-        while (
-            self.memory.task_stack_peek() and self.memory.task_stack_peek().task.check_finished()
-        ):
-            self.memory.task_stack_pop()
-
-        # If nothing to do, wait a moment
-        if self.memory.task_stack_peek() is None:
+        query = {"base_table": "Tasks", "base_range": {"minprio": -0.5, "maxpaused": 0.5}}
```

Comment:
we need a documentation describing what these numbers and keys mean and what they are doing